.gitmodules

```diff
@@ -220,3 +220,9 @@
 [submodule "projects/braincode_Epilepsy_Priority_Setting_Partnership"]
 	path = projects/braincode_Epilepsy_Priority_Setting_Partnership
 	url = https://github.com/conpdatasets/braincode_Epilepsy_Priority_Setting_Partnership
+[submodule "projects/braincode_CP-NET"]
+	path = projects/braincode_CP-NET
+	url = https://github.com/CONP-PCNO/braincode_CP-NET
+[submodule "projects/braincode_EpLink"]
+	path = projects/braincode_EpLink
+	url = https://github.com/CONP-PCNO/braincode_EpLink
```

Comment:
Same as for CP-NET: the README looks good and the overview is the same as the description field in the DATS.json.